apps/mobile/src/features/Send/useCases/ListAmountsToSend/AddToken/SelectTokenFromListScreen.tsx

```diff
@@ -360,12 +361,7 @@ const NoSpendableAmount = ({text}: {text: string}) => {
 
       <Space.Height.lg />
 
-      <Text
-        style={[
-          a.heading_3_medium,
-          {color: p.gray_max, flex: 1, textAlign: 'center'},
-        ]}
-      >
+      <Text style={[a.heading_3_medium, {color: p.gray_max, flex: 1}]}>
```

Comment:
I think all p.gray_max needs to be moved to some ta.text_ something, need to check with design
```suggestion
      <Text style={[a.heading_3_medium, {color: p.gray_max}, a.flex_1]}>
```

Reply:
There's an element one maybe?

Reply:
there are some text_...